### DISCUSSION TITLE
Actions Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#48778] [Actions]
Hello and welcome to another check-in. We’re back with the latest from the Actions world. 🌎 This week, I’ll share some of the new additions to Actions, thank you's to our incredible community members, and a opportunity to share your valuable feedback.

Wait! Did you notice something has changed? 💭 

[GitHub Actions](https://github.com/orgs/community/discussions/categories/actions) and [GitHub Packages](https://github.com/orgs/community/discussions/categories/packages) are now two separate categories. Since both GitHub Actions and GitHub Packages are separate products, we wanted to help you browse relevant discussions and post your questions in specific product categories. With this change, I will be focusing on Actions and my colleague [@otter-computer](https://github.com/otter-computer) will lean more into the Packages category. Based on your feedback and frequently asked questions in the community, we’ve pinned the [Actions FAQs](https://github.com/orgs/community/discussions/48283) and the [Packages FAQs](https://github.com/orgs/community/discussions/48280) in their specific categories.

I will continue to post these check-ins bi-weekly to share all the hot & exciting Actions updates that you don’t want to miss out on. Let’s get started! 🏁 

---

## 💌  From the Community - Shoutout 📢  to these members

- Great solution ✅ from [@kenjitagawa](https://github.com/kenjitagawa) solving an issue for [@kamillaallimak](https://github.com/kamillaallimak) where [Custom Domain was showing a blank page](https://github.com/orgs/community/discussions/48138)
- [@gforcada](https://github.com/gforcada) asked a tricky [question](https://github.com/orgs/community/discussions/46785) but our champion [@ChristopherHX](https://github.com/ChristopherHX) provided the perfect answer. 👏
- A big round of applause 👏  for these Self-Solvers 
  - [@JustinBuiel](https://github.com/JustinBuiel) demonstrated how a [perfect self-solve](https://github.com/orgs/community/discussions/47930) should look like.They didn’t just updated the discussion but also shared the steps to solve the issue. Discussions like these really help other community members to easily get the answers they need. 💯
  - [@jkruse14](https://github.com/jkruse14) got the [runner authentication](https://github.com/orgs/community/discussions/47563) sorted after thoroughly trying and testing their script. Thank you for coming back to the discussion and updating it.❤️

---

## [ Latest Ships 🚢 in Actions](https://github.com/orgs/github/projects/4247) -  Improved Larger Hosted Runners 🧑‍💻

Actions users of GitHub-hosted larger Linux runners will be able to make use of [hardware acceleration for Android testing](https://github.blog/changelog/2023-02-23-hardware-accelerated-android-virtualization-on-actions-windows-and-linux-larger-hosted-runners/). Wanted to highlight that this change was made possible based on the great feedback provided by [@gsauthof](https://github.com/gsauthof) 👏

In addition to this change, users of GitHub-hosted larger runners will no longer be able to add, edit or remove additional labels on existing or new runners. [Removal of additional label option for GitHub-hosted larger runners](https://github.blog/changelog/2023-02-15-github-actions-removal-of-additional-label-option-for-github-hosted-larger-runners/) provides a more deterministic approach to make the runner management easier for you.

Do you wanna learn more about GitHub Actions and build strong fundamentals? Join GitHub and other industry experts on how you can use GitHub Actions to better understand dependencies, identify and fix bottlenecks, and ship quality code faster.

[Register Now](https://resources.github.com/devops/ci-cd-with-github-actions/) - March 9 📅 from 10:00 am to 12:00 pm PST (Webinar)
 
---

## Your Feedback Matters 🤝 

We want to hear 👂  about your user experience with posting a new discussion in the community. We're exploring ways to gauge your experience with GitHub community on Discussions and want to ensure that you have the right information to ask questions effectively, report an issue, and share your feedback easily. This is what the current “Start a new discussion” section looks like in the Actions category:

![Screenshot 2023-02-24 at 3 47 38 PM](https://user-images.githubusercontent.com/95834130/221163447-90947693-ed35-4fd9-b68e-ef9d01e63125.png)

- When sharing your feedback or requesting a new feature, which Topic (under "Select Topic area") do you relate better to - Product Feedback or Question or a new topic as a “Feature Request”?

- Do you find it easier to decide which category you need to post your question or feedback into? Does the current description for Actions as a product relate to you well enough?

- Do you label your discussion using the `Labels` (on the right) before posting it? If not, why so and how can we make it easier for you?

These are just a few questions on top of my mind. I’ll be around to make this experience even better for you, so if you have any suggestions please post them in the comments below. With that, it’s a wrap for this week's Actions check-in. Till next time! 🚀

**Comment by jsoref** [comment]
Currently [`github.action_repository` appears incredibly broken](https://github.com/community/community/discussions/49245). 

If this could be fixed in time for the next round up, that'd be great.

Also, if someone could actually connect with me so that I can talk about the various parts of the github.com/actions/* repository ecosystem that are not working well, I'd appreciate it.

**Reply by Akash1134** [reply]
Hi @jsoref Please share your valuable feedback via a discussion post in the [Actions category](https://github.com/orgs/community/discussions/categories/actions). I'll be sure to pass it on to the product team so that they can review it. Thanks a lot for your contributions to the GitHub community. 🙂
